org.aposin.licensescout.core/launch/org.aposin.licensescout.core_clean_deploy.launch

```diff
@@ -1,21 +1,21 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<launchConfiguration type="org.eclipse.m2e.Maven2LaunchConfigurationType">

```

Comment:
This indentation changes hide the real modification - could you revert where there is no change in the line?

Reply:
The file had the wrong line endings (Windows). I fixed it to Unix with this change.

Reply:
Looks like the problem is still here could you please double check?

Reply:
Regarding indentation, see below.